packages/adapters-library/src/adapters/beefy/sdk/config.ts

```diff
@@ -18,7 +18,7 @@ import { fetchIchiBalanceBreakdown } from './unwrap/ichi'
 import { fetchPendleEquilibriaBalanceBreakdown } from './unwrap/pendle_equilibria'
 import { fetchSolidlyBalanceBreakdown } from './unwrap/solidly'
 
-export const chainIdMap: Record<Chain, string> = {
+export const chainIdMap: Partial<Record<Chain, string>> = {
```

Comment:
If set to `EvmChain` it'll require changing `Chain` to `EvmChain` in every Beefy adapter using it. This is way easier.